core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -148,6 +152,59 @@ private[spark] class EventLoggingListener(
     }
   }
 
+  /** cut the event log file **/
+  private def logReload(): Unit = {
```

Comment:
Is it consistent? This function name is `logReload` and the description is `cut the event log file`. It seems to be opposite (read vs write).